packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -296,7 +296,10 @@ export class CdkToolkit {
    * OUTPUT: If more than one stack ends up being selected, an output directory
    * should be supplied, where the templates will be written.
    */
-  public async synth(stackNames: string[], exclusively: boolean, quiet: boolean): Promise<any> {
+  public async synth(stackNames: string[], exclusively?: boolean, quiet?: boolean): Promise<any> {
```

Comment:
I made `exclusively` and `quiet` optional so that I could pass in `undefined` in the new unit test to replicate the default behavior when the user does not specify those parameters. Definitely open to ideas for better ways to imitate that scenario.

Reply:
I think I'd prefer to have made this change in `cdk.ts` where the command line arguments get parsed and then passed to this function.

Is there a reason you decided to do it here instead of there?